modules/material/themes/material/silauth/loginuserpass.twig

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="{{ currentLanguage }}">
+<head>
+  <title>{{ '{login:title}'|trans({'%idpName%': idpName}) }}</title>
+
+  {% include 'header.twig' %}
+
+  <script>
+    function preventDefault(event) {
+      event.preventDefault();
+    }
+  </script>
+
+  {% if siteKey is defined %}
```

Comment:
I am not sure if defined would catch an empty, but still defined siteKey.

```suggestion
  {% if siteKey is not empty %}
```

Reply:
Tested this with fiddle, I was correct. Should be changed.

Reply:
It has to check 'defined' or have a null coalesce since `siteKey` will not be defined if the out-of-the-box SSP controller is used. Eventually it won't be, but I'm not there yet.